src/App.js

```diff
@@ -1,11 +1,15 @@
 import React from 'react';
 import NavBar from "./Components/Navbar";
+import {StylesProvider} from "@material-ui/core/styles";
 
 function App() {
     return (
         <div className="App">
-            <NavBar/>
+            <StylesProvider injectFirst>
```

Comment:
Just asking to know!
Do we need `<StylesProvider>` to be able to use `styled-components`?

Reply:
Yes.it injects style for components of Matrial-Ui library.